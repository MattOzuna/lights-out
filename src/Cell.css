.Cell {
  height: 100px;
  width: 100px;
  border-radius: 5px;
  background-color: rgb(66, 65, 65);
  cursor: pointer;
}

.Cell-lit {
  background-color: rgb(229, 237, 8);
}