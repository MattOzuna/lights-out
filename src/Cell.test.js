import React from "react";
import Cell from "./Cell";
import {render, asFragment} from "@testing-library/react"

it('should render properly', ()=>{
    render(
        <table>
            <tbody>
                <tr>
                    <Cell />
                </tr>
            </tbody>
        </table>
        )
})

it('should match snapshot of a lit Cell', ()=>{
    const {asFragment} = render(        
    <table>
        <tbody>
            <tr>
                <Cell isLit={true}/>
            </tr>
        </tbody>
    </table>)
    expect(asFragment()).toMatchSnapshot()
})

it('should match snapshot of a not lit Cell', ()=>{
    const {asFragment} = render(        
    <table>
        <tbody>
            <tr>
                <Cell isLit={false}/>
            </tr>
        </tbody>
    </table>)
    expect(asFragment()).toMatchSnapshot()
})



