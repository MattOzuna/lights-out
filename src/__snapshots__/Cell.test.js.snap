// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot of a lit Cell 1`] = `
<DocumentFragment>
  <table>
    <tbody>
      <tr>
        <td
          class="Cell Cell-lit"
        />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;

exports[`should match snapshot of a not lit Cell 1`] = `
<DocumentFragment>
  <table>
    <tbody>
      <tr>
        <td
          class="Cell "
        />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
